analytics/population/README.md

```diff
@@ -0,0 +1,30 @@
+<div style = "text-align: justify"> 
+  
+# **Updated population density of the LA city neighborhood councils (NCs).**
+  
+### Have calculated the population of LA city neighborhood councils using the updated NC shape file, Census tracts data-2020 and 2020-ACS (5 year estimate) demographics data. The notebook- NC_pop_recent.ipynb outlines the detailed steps involved to evaluate the population, area (in square miles) and population density of the 99 NCs. 
+
+### The movtivation behind this analysis stems from the fact that the latest available population data for the NCs have been estimated using older census population/demographics data approximated from block groups to LA NCs (97 NCs). The boundary of NC has been modified around 2018- adding 2 more NCs- Arts District Little Tokyo NC (also known as Historic Cultural) and Westwood NC. 
+
+### How to use this tool:
+- #### NC_pop_recent.ipynb- This notebook is an updated version of NC_pop_recent_no_filter.ipynb, where in, area and population filter have been introduced to address the inflated LA city NCs value in comparison to the one reported in the Census Bureau [website](https://www.census.gov/quickfacts/losangelescitycalifornia?).
```

Comment:
Please remove reference to NC_pop_recent_no_filter.ipynb since that has been removed

Reply:
Removed NC_pop_recent_no_filter.ipynb.